doc/sphinx/proofs/writing-proofs/inductive.rst

```diff
@@ -1110,13 +1108,15 @@ symbol :g:`=`.
 
 .. tacv:: simplify_eq
 
-   If the current goal has form :g:`t1 <> t2`, it behaves as
+   If the current goal has form :g:`t1 <> t2`, it behaves like
    :n:`intro @ident; simplify_eq @ident`.
 
-.. tacn:: dependent rewrite -> @ident
+.. tacn:: dependent rewrite {? {| -> | <- } } @one_term {? in @ident }
    :name: dependent rewrite ->
 
-   This tactic applies to any goal. If :n:`@ident` has type
+   .. todo PR: what is "existT" below?
```

Comment:
The constructor of `sigT`, defined in the standard library

Reply:
OK, thanks.  Still mysterious, but I'll let it go for now.

Off topic: I've noticed that it's difficult to find the definitions of things like `sigT` and `existT` with a global text search because people reuse the identifier.  How useful would it be if you could shift-click on any identifier or notation symbol in the GUI to jump to its definition?  Or to search for all uses of a definition in a given file or perhaps globally?

Reply:
https://github.com/coq/coq/blob/e60305c12305d298fe7e36738b8c95eca3d7409e/theories/Init/Specif.v#L43-L45



> How useful would it be if you could shift-click on any identifier or notation symbol in the GUI to jump to its definition? Or to search for all uses of a definition in a given file or perhaps globally?

Those both sound useful, though I personally primarily use ProofGeneral.  I think another useful thing would be links between the refman and the stdlib.

Reply:
For jumping to the definition, you'd do most of the work on the Coq side so it can be shared among multiple GUIs. So should be easy for PG to support it.